src/System.Security.Cryptography.X509Certificates/tests/X509FilesystemTests.Unix.cs

```diff
@@ -15,12 +15,14 @@ namespace System.Security.Cryptography.X509Certificates.Tests
     [Collection("X509Filesystem")]
     public static class X509FilesystemTests
     {
-        // #9293: Our Fedora and Ubuntu1610 CI machines use NTFS for "tmphome", which causes our filesystem permissions checks to fail.
-        private static bool IsReliableInCI { get; } = ChainTests.IsReliableInCI;
+        private static bool CanModifyStores { get; } = TestEnvironmentConfiguration.CanModifyStores;
+        private static bool RunManualTests { get; } = TestEnvironmentConfiguration.RunManualTests;
 
-        [ActiveIssue(12833, TestPlatforms.AnyUnix)]
-        [Fact]
         [OuterLoop]
+        // This test is a bit too flaky to be on in the normal run, even for OuterLoop.
+        // It can fail due to networking problems, and due to the filesystem interactions it doesn't
+        // have strong isolation from other tests (even in different processes).
+        [ConditionalFact(nameof(RunManualTests))]
```

Comment:
We can go with it for now.  We just may want to re-review at some point to determine whether they're actually being used.